egs/uw3/v1/local/score.sh

```diff
@@ -22,6 +22,8 @@ if [ $# -ne 3 ]; then
   echo " Options:"
```

Comment:
Jonathan, I guess you could have a simple script with only 3 lines:
```
#!/bin/bash
steps/scoring/score_kaldi_wer.sh "$@"                                                                                                                                 
steps/scoring/score_kaldi_cer.sh --stage 2 "$@" 
```
Please also test it before updating.